src/charts/DownloadWrapper.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useRef } from 'react';
+
+import { exportComponentAsPNG } from 'react-component-export-image';
+import { FaCloudDownloadAlt } from 'react-icons/fa';
+import { BiTable } from 'react-icons/bi';
+import ReactTooltip from 'react-tooltip';
+import styled, { css } from 'styled-components';
+import { CSVLink } from 'react-csv';
+
+const BUTTON_SIZE = '2.5em';
+
+const baseButtonStyles = css`
+  position: absolute;
+  z-index: 10;
+  top: 0px;
+  right: 8px;
+  padding: 8px 8px 8px 8px;
+  transform: translate(0%, 0%);
+  transition: 0.1s ease-in;
+  &:hover {
+    cursor: pointer;
+    transform: translate(0%, -10%);
+    transition: 0.2s ease-out;
+  }
+`;
+
+const DownloadButton = styled(FaCloudDownloadAlt)`
+  ${baseButtonStyles}
+`;
+const DownloadDataButton = styled(BiTable)`
+  ${baseButtonStyles}
+  right: 38px;
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+  height: 100%;
+`;
+
+const DownloadContainer = styled.div`
+  position: relative;
+  height: 100%;
+`;
+
+//Adds button to download wrapper component as an image
+const DownloadWrapper = ({
+  name = 'plot',
+  rawData = [],
+  dataProcessor = (data: any[]) => data,
+  ...props
+}: any) => {
+  const componentRef = useRef(null);
+
+  const exportOptions = {
+    fileName: name + '.png',
+    html2CanvasOptions: {
+      scale: 8,
+    },
+  };
+
+  //keep for debug
```

Comment:
Please don't keep commented out code around for debugging. It often never gets used again. You can stash this code if you want to keep it for yourself.